app/lib/features/account_deletion_config.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Features
+  class AccountDeletionConfig
```

Comment:
This whole file is exactly the same as we had in `app/lib/account_deletion_config.rb` but in `app/lib/features/account_deletion_config.rb` and inside the module `Features`